app/models/user.rb

```diff
@@ -1,8 +1,18 @@
 class User < ApplicationRecord
   has_secure_password
+  after_initialize :set_role
 
   has_one :subscription
-  validates :name, :address, presence: true
+  validates :name, :address, :role, presence: true
   validates :email, uniqueness: true, presence: true
   validates :password_digest, presence: true
+  enum role: %w(guest user admin)
+
+  def set_role
```

Comment:
I am setting default values after the user action is created, it's easier to change the default value here instead of having to generate migrations in the future